tests/unit_tests/gui/test_main_window.py

```diff
@@ -237,24 +223,11 @@ def test_that_run_dialog_can_be_closed_after_used_to_open_plots(
     with StorageService.init_service(
         project=os.path.abspath(ert_config.ens_path),
     ):
-        gui = _setup_main_window(enkf_main, args_mock, GUILogHandler())
-        gui.notifier.set_storage(storage)
+        gui = _setup_main_window(enkf_main, args_mock, GUILogHandler(), storage)
         qtbot.addWidget(gui)
         simulation_mode = get_child(gui, QComboBox, name="Simulation_mode")
         start_simulation = get_child(gui, QToolButton, name="start_simulation")
 
-        def handle_dialog():
-            message_box = wait_for_child(gui, qtbot, QMessageBox)
-            qtbot.mouseClick(message_box.button(QMessageBox.Yes), Qt.LeftButton)
-
-            QTimer.singleShot(
-                500,
-                lambda: handle_run_path_dialog(
-                    gui=gui, qtbot=qtbot, delete_run_path=False
-                ),
-            )
-
```

Comment:
This test might be hanging because in github actions the 'existing run path dialog' is shown? 🤷🏻